docs(aio): update migrated content from anguar.io

petebacondarwin opened the pull request.

wardbell reviewed (approved):
This is a necessary change that does not require a file by file review (fortunately). The Travis failure is confined to `aio`-land is something to be dealt with later

petebacondarwin commented:
The build failure is actually due to an invalid doc region marker in an angular.io example.
See https://github.com/angular/angular.io/blob/15036913bb102b561a376d36164ca8a9054dc96f/public/docs/_examples/toh-3/ts/src/app/hero-detail.component.ts#L28

petebacondarwin commented:
The failure on Travis is now because the content expects #15425 to have landed.

petebacondarwin committed:
build(aio): update Rho to a version that accepts custom-elements

petebacondarwin committed:
docs(aio): update migrated content from anguar.io

petebacondarwin committed:
build(aio): include top level docs in content

petebacondarwin committed:
build(aio): render ALL code-examples, not just the first

petebacondarwin committed:
test(aio): the tutorial link no longer ends with a slash

And since the navigation opens to the initial page, we don't need
to click the link.

mary-poppins commented:
The angular.io preview for 23d363c is available [here][1].

[1]: https://pr15462-23d363cd45058610b516dd0ef76c1af25b987ce6.ngbuilds.io/

petebacondarwin committed:
fix(aio): ensure DocumentService doesn't crash on bad JSON

mary-poppins commented:
The angular.io preview for 17ae829 is available [here][1].

[1]: https://pr15462-17ae829aeadc22890896b70b25cc91b4138fcce8.ngbuilds.io/
